frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -61,18 +59,16 @@ const makeOCSRequest = (
   selectedData: NodeKind[],
   storageClass: StorageClassResourceKind,
   isEncrypted: boolean,
-  isMinimal?: boolean,
-  isEncryptionSupported?: boolean,
+  isMinimal: boolean,
 ): Promise<any> => {
-  const promises = makeLabelNodesRequest(selectedData);
+  const promises = labelNodes(selectedData);
```

Comment:
```suggestion
  const promises = labelNodes(selectedNodes);
```
Let's rename this while we are at it.